phpunit.xml.dist

```diff
@@ -5,7 +5,6 @@
          convertWarningsToExceptions="true"
          beStrictAboutOutputDuringTests="true"
          beStrictAboutTestsThatDoNotTestAnything="true"
-         forceCoversAnnotation="true"
```

Comment:
Because some tests don't have the `@cover` annotation (and the latest PHPUnit doesn't like this).